packages/core/src/utils/validate-sign-in-experience.ts

```diff
@@ -37,5 +55,19 @@ export const validateSignInMethods = (signInMethods?: SignInMethods) => {
     'sign_in_experiences.not_one_and_only_one_primary_sign_in_method'
   );
 
-  // TODO: assert others next PR
+  const enabledConnectorInstances = await enabledConnectorInstancesPromise;
+
+  if (isEnabled(signInMethods.email)) {
+    assertEnabledConnectorByType(ConnectorType.Email, enabledConnectorInstances);
```

Comment:
Can we just write `assertThat`? Is just a single line.

Reply:
There are 3 places to use this logic, so extract it.

Reply:
Inlined. PTAL.